tests/integration/bootstrap-system.test.js

```
@@ -0,0 +1,777 @@
+        const exitCode = await bootstrap.run();
+        expect(exitCode).toBe(0);
+
+        const duration = Date.now() - startTime;
+        runTimes.push(duration);
+
+        // Clean up for next iteration
+        await bootstrap.dbHelpers?.cleanup();
+
+        // Create new bootstrap instance for next run
+        if (i < 2) {
+          bootstrap = new BootstrapSystem();
```

Biome (2.1.2)

[error] Can't assign bootstrap because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)